### PR TITLE
SIGN-7090 - updated changelist and version to setup automatic release process

### PR DESCRIPTION
**Pull request by volbobvol**
SIGN-7090 - updated changelist and version in the POM file to setup automatic release process

### Submitter checklist
- [ x ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ x ] Ensure that the pull request title represents the desired changelog entry
- [ x ] Please describe what you did
- [ x ] Link to relevant issues in GitHub or Jira

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -9,11 +9,11 @@
     </parent>
     <groupId>io.jenkins.plugins</groupId>
     <artifactId>signpath</artifactId>
-    <version>${revision}${changelist}</version>
+    <version>${changelist}</version>
     <packaging>hpi</packaging>
     <properties>
         <revision>2.0.1</revision>
```
Comment (timja) [inline pom.xml:15]: this should also be removed as you are no longer using it
```suggestion
```

File: pom.xml [hunk 2]
```diff
@@ -9,11 +9,11 @@
...
-        <changelist></changelist>
+        <changelist>999999-SNAPSHOT</changelist>
         <jenkins.version>2.359</jenkins.version>
```
Comment (timja) [inline pom.xml:17]: you shouldn't depend on random weekly versions long term.
the current recommended version is:

```suggestion
        <jenkins.baseline>2.452</jenkins.baseline> 
        <jenkins.version>${jenkins.baseline}.4</jenkins.version>
```

and then update the bom version at the bottom of the file to `<artifactId>bom-${jenkins.baseline}.x</artifactId>`
and version to https://github.com/jenkinsci/bom/releases/tag/3276.vcd71db_867fb_2

File: pom.xml [hunk 3]
```diff
@@ -9,11 +9,11 @@
...
         <java.level>11</java.level>
```
Comment (timja) [inline pom.xml:18]: these shouldn't be overridden unless you really know what you are doing. 
line 18-22 I think could probably be deleted

**Review by timja — APPROVED** [review]
I pushed another commit to fixup something else but LGTM

File: pom.xml (hunk 3 above)
Comment (volbobvol) [reply pom.xml:19]: I pushed one more commit here 
https://github.com/jenkinsci/signpath-plugin/pull/25/commits/ba9ad3d04e7af9b694c02c803bf049de40e6cb0f

The goal is to have a manually controlled version prefix and auto-increment for the build number.

**Pull request merged**